.tugboat/config.yml

```diff
@@ -1,11 +1,12 @@
 services:
   php:
-    image: tugboatqa/php:7.2-apache
+    image: tugboatqa/php:8.1-apache
     default: true
     depends: mariadb
     commands:
       init:
         # Install PHP 'zip' extension to allow installing Backdrop modules.
+        - apt update && apt install -y libzip-dev
```

Comment:
Interesting, why do we need libzip-dev with 8.1?

Reply:
I've noticed this too in some of the newer PHP images (I saw it in at least 7.4). This package is necessary in order for PHP to install the zip extension. I think it's the dev package because it actually recompiles PHP to include the zip library. I have used this same line (or similar) on multiple recent projects. Seems good to me.